src/tqec/compile/detectors.py

```diff
@@ -203,14 +203,19 @@ def _compute_detectors_for_fixed_radius(
         for j, subtemplate_index in enumerate(row):
             if subtemplate_index == 0:
                 continue
-            plaquette_origin = Position(j * increments.x, i * increments.x)
+            plaquette_origin = Position2D(j * increments.y, i * increments.x)
             for d in detectors_by_subtemplate[subtemplate_index]:
                 offset_measurements = list(
                     m.offset_spatially_by(plaquette_origin.x, plaquette_origin.y)
                     for m in d.measurement_data
                 )
+                coordinates = (
+                    d.coordinates[0] + plaquette_origin.y,
+                    d.coordinates[1] + plaquette_origin.x,
```

Comment:
A long-standing question I had is about this particular piece of code. 

Should we output detectors coordinates using `stim`'s coordinate system (that seems to be in `(y, x)` like you implemented) or in our coordinate system:

```suggestion
                    d.coordinates[0] + plaquette_origin.x,
                    d.coordinates[1] + plaquette_origin.y,
```

I kind of agree with using `stim`'s coordinate system, else everything becomes hard to read for sure (e.g., "is this in `(x, y)` or `(y, x)` coordinate?), but then a comment explaining why this is done would be welcomed.

Reply:
This is not `stim`'s coordinate system but `cirq`'s(`cirq.GridQubit`). As we are translating `cirq.Circuit` to `stim.Circuit`, I try to make sure the coordinates of the detectors align with the qubits.